src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -321,6 +321,11 @@ public enum EvalMetricType
         [TGUI(Label = "Ranking Label Gain")]
         public string CustomGains = "0,3,7,15,31,63,127,255,511,1023,2047,4095";
 
+        [Argument(ArgumentType.AtMostOnce, HelpText = "Parameter for the sigmoid function. Used only in binary and multiclassova " +
+            "classification and in lambdarank applications", ShortName = "sigmoid")]
+        [TGUI(Label = "Sigmoid")]
+        public double Sigmoid = 1;
```

Comment:
If the hyperparameter has a notice effect, and the user can generally get gains by changing it, we should add a sweep range to the hyperparameter. E.g.: https://github.com/dotnet/machinelearning/blob/f2e54852d2d2a77180c3eee75317b93c683ef6c9/src/Microsoft.ML.LightGBM/LightGbmArguments.cs#L135-137

Reply:
Do we have a sweep range for sigmoid that you are aware of?

Reply:
Do you mean between 0 and 1? (not including zero). @guolinke your thoughts?

Reply:
That's a good question. 

As we had a fixed value before, I haven't had a chance to explore what a good range looks like. Values >0 is acceptable, but I don't know "good" ranges; for instance is 10 a useful value? 500? Or should we focus (0..1]?

Reply:
Not sure about the effect of sigmoid. smaller sigmoid somehow like the using smaller learning rate. So I think using learning_rate searching may is enough.